File: npm/src/commands/status.js
'use strict'
const { Command } = require('commander')
const { getStatus } = require('../validator')
const { t } = require('../i18n')

const cmd = new Command('status')
cmd.description(t('status.description'))
cmd.action(async () => {
  console.log(await getStatus())
})

module.exports = cmd

File: internal/commands/status.go
package commands

import (
	"fmt"

	"github.com/spf13/cobra"
	"github.com/kgsaran/trackfw/internal/validator"
)

func newStatusCmd() *cobra.Command {
	return &cobra.Command{
		Use:   "status",
		Short: "Show current governance state (wip, blocked, recent done)",
		RunE: func(cmd *cobra.Command, args []string) error {
			state, err := validator.GetStatus()
			if err != nil {
				return err
			}
			fmt.Print(state)
			return nil
		},
	}
}
